kuka_fortec_support/config/fortec_urdf.rviz

```diff
@@ -0,0 +1,251 @@
+Panels:
```

Comment:
the rviz confi file is the same for all 6-dof robots and was moved to the `kuka_resources` package (`config/view_6_axis_urdf.rviz`), so I think it is not needed here

Reply:
Hi @Svastits, yes of course I noticed this, but I did quite a bit of work to convert to .dae, clean and make the visual robot look nice, so I wanted to show it off 😉

I will revert this to using the common resource to avoid repeat, but pls consider my .dae example instead of the transparent red standard rviz config for the common pkg. I also offer my help to cleanup other robots' meshes in the same way, and add collisions, if you want.

Reply:
Hi @Pugens I also noticed, that the rviz config in the `kuka_resources` does not look quite okay, so I modified it to hide the collision and disable transparency. (The red boxes were the collision geometry, so the visual also looks nice now in my opinion.) 

For the collision geometry I have found that the Blender remesh modifier can reduce planning times the most compared with Convex Hull and Decimate modifier (I have only run a few tests with a specific robot, `lbr_iisy3_r760`, but the difference was quite big). I plan to automate this remeshing using the Blender python API. What is your experience with collision geometry, do you think there is a most efficient way of doing this?

I have no experience in creating dae files (yours are really nice indeed!), I would appreciate if you could help with those in some way :)

Reply:
@Svastits thanks for the comment! I added your changes and fixed any new bugs due to them. I should have gotten all of them!

Next time I will be working on this repo will be for either gazebo stuff, or for the KUKA kr210 meshes :)
I have some pretty awesome material to share...

Regarding the collision, I also think just the convex hull from blender is disappointing. I usually do it with a decimate anyways tho, as the re-mesh does not really get out there to all the points sticking out!

![image](https://github.com/kroshu/kuka_robot_descriptions/assets/19688353/f665bf5a-3b98-49e2-9055-819765668b08)

Reply:
Thanks for your contribution :)
These meshes indeed look awesome, we are looking forward to seeing you here again ;)